nodepool/nodepool.yaml

```diff
@@ -31,6 +31,12 @@ labels:
     min-ready: 4
```

Comment:
Consider bumping this down, because 30 total is about where it started bogging down the disk I/O of all the compute nodes. No point running 4 you don't use.